### PR TITLE
Fixes #30 -  documentation for labels

### LINKED ISSUE #30: add filter label documentation

### PR DESCRIPTION
**Pull request by madeleine-neumann**
Adds the documentation and code examples for labels

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -1353,6 +1353,27 @@ <h4 class="headline-3">Module: Button</h4>
 		</div>
 	</section>
 
+	<section class="grid">
+		<div class="grid-row">
+			<div class="grid-cell x3 hint-text">
+				<h4 class="headline-3">Module: Issue Labels</h4>
+				<p>
+					This Module helps you to set filter labels. The ID <code>filter-issue-status</code> wraps the labels. The <code>filter-label</code> class makes sure, that all labels are wrapped in a container with a 100% width and the text is centered. The current used label gets the class <code>active</code> as well to show in wich state the ticket is currently.
```
Comment (zoepage) [inline docs/index.html]: This module helps to add a label to the issue, which indicates the status of an issue. 
There are 6 different states.

- Needs Triage 
- Needs Diagnosis
- Needs Contact
- Ready for Outreach
- Site Contacted
- Closed

File: docs/index.html [hunk 2]
```diff
@@ -1353,6 +1353,27 @@ <h4 class="headline-3">Module: Button</h4>
...
 	</section>
 
+	<section class="grid">
+		<div class="grid-row">
+			<div class="grid-cell x3 hint-text">
+				<h4 class="headline-3">Module: Issue Labels</h4>
+				<p>
+					This Module helps you to set filter labels. The ID <code>filter-issue-status</code> wraps the labels. The <code>filter-label</code> class makes sure, that all labels are wrapped in a container with a 100% width and the text is centered. The current used label gets the class <code>active</code> as well to show in wich state the ticket is currently. 
+				</p>
+				<p>There are 6 labels you can use:</p>
```
Comment (zoepage) [inline docs/index.html]: this and the next paragraph can be removed as we have the context above. 
(first context, then code explanation).

**Review by zoepage — CHANGES_REQUESTED** [review]

**Comment by madeleine-neumann** [comment]
@zoepage another review?

**Comment by zoepage** [comment]
@madeleine-neumann merge conflicts?

**Comment by madeleine-neumann** [comment]
@zoepage done

**Review by zoepage — APPROVED** [review]

**Pull request merged**